### PR TITLE
Fix up File & Data Service docs

### PR DESCRIPTION
**Pull request by ada-cienciala**
Found some typos, grammar errors and redundant lines while reading the documentation. Decided to chip in.

Left some comments in places where there was something missing, like examples of code or descriptions. To be deleted once addressed.

### CODE REVIEW

File: docs/services/manage-data/data-service/schemas.md [hunk 1]
```diff
@@ -615,15 +617,14 @@ await sdk.data.transitions.create(newSchema.id, {
...
-
-{% endtab %}
 {% endtabs %}
 
 ## Indexes
 
 The Index object is identified by an id and a name. An index is set on a specific property in a Schema. This property is defined in the Fields object by the name and type attribute. The index is tailored with the following attributes:
 
+<!-- Some attributes are not described -->
+
```
Comment (ada-cienciala) [inline docs/services/manage-data/data-service/schemas.md:627]: The background and system attributes are not described properly. 

To be discarded once addressed.

File: docs/services/manage-data/data-service/schemas.md [hunk 2]
```diff
@@ -615,15 +617,14 @@ await sdk.data.transitions.create(newSchema.id, {
 ```
 {% endtab %}
 
-{% tab title="Second Tab" %}
-
-{% endtab %}
```
Comment (ada-cienciala) [inline docs/services/manage-data/data-service/schemas.md:620]: There was no second example

File: docs/services/manage-data/data-service/schemas.md [hunk 3]
```diff
@@ -454,6 +454,8 @@ When executing a transition on a document you can require the API client to prov
 the example above would require the API client to provide the address.inhabited property. otherwise the transition will not trigger.
 {% endtab %}
 
+<!-- Remaining examples are redundant or not filled out -->
+
```
Comment (ada-cienciala) [inline docs/services/manage-data/data-service/schemas.md:458]: The `document ` example is copied from `inputCondition` tab and is incorrect. `initiatorHasRelationToUserInData` and `initiatorHasRelationToGroupInData` are empty so could be deleted or filled out.

To be discarded once addressed.

**Review by wwaaijer-exh — APPROVED** [review]
Thank you very much! 🙏

I've created tickets in our internal issue tracker for the extra comments you've added. We'll definitely take a look at those inconsistencies.

**Pull request merged**